--- a/browser/locales/en-US/browser/preferences/preferences.ftl
+++ b/browser/locales/en-US/browser/preferences/preferences.ftl
@@ -142,6 +142,8 @@ search-results-help-link = Need help? Visit <a data-l10n-name="url">{ -brand-sho
 
 startup-header = Startup
 
+session-restore-learn-more = What if it does not work?
+
 always-check-default =
     .label = Always check if { -brand-short-name } is your default browser
     .accesskey = y
@@ -489,7 +491,7 @@ play-drm-content =
     .label = Play DRM-controlled content
     .accesskey = P
 
-play-drm-content-learn-more = Learn more
+play-drm-content-learn-more = Why we disable it
 
 update-application-title = { -brand-short-name } Updates
 
@@ -497,7 +499,7 @@ update-application-description = Keep { -brand-short-name } up to date for the b
 
 # Variables:
 # $version (string) - Firefox version
-update-application-version = Version { $version } <a data-l10n-name="learn-more">What’s new</a>
+update-application-version = Version { $version } <a data-l10n-name="learn-more">Visit the repositories</a>
 
 update-history =
     .label = Show Update History…
@@ -664,7 +666,7 @@ home-restore-defaults =
     .accesskey = R
 
 home-mode-choice-default-fx =
-    .label = { -firefox-home-brand-name } (Default)
+    .label = Homepage (Default)
 
 home-mode-choice-custom =
     .label = Custom URLs…
@@ -693,8 +695,8 @@ choose-bookmark =
 
 ## Home Section - Firefox Home Content Customization
 
-home-prefs-content-header2 = { -firefox-home-brand-name } Content
-home-prefs-content-description2 = Choose what content you want on your { -firefox-home-brand-name } screen.
+home-prefs-content-header2 = Homepage Content
+home-prefs-content-description2 = Choose what content you want on your Homepage screen.
 
 home-prefs-search-header =
     .label = Web Search
