--- a/browser/locales/en-US/browser/preferences/preferences.ftl
+++ b/browser/locales/en-US/browser/preferences/preferences.ftl
@@ -128,6 +128,8 @@ search-results-help-link = Need help? Visit <a data-l10n-name="url">{ -brand-sho
 
 startup-header = Startup
 
+session-restore-learn-more = What if it does not work?
+
 always-check-default =
     .label = Always check if { -brand-short-name } is your default browser
     .accesskey = y
@@ -472,7 +474,7 @@ play-drm-content =
     .label = Play DRM-controlled content
     .accesskey = P
 
-play-drm-content-learn-more = Learn more
+play-drm-content-learn-more = Why we disable it
 
 update-application-title = { -brand-short-name } Updates
 
@@ -480,7 +482,7 @@ update-application-description = Keep { -brand-short-name } up to date for the b
 
 # Variables:
 # $version (string) - Firefox version
-update-application-version = Version { $version } <a data-l10n-name="learn-more">What’s new</a>
+update-application-version = Version { $version } <a data-l10n-name="learn-more">Visit the repositories</a>
 
 update-history =
     .label = Show Update History…
@@ -639,7 +641,7 @@ home-restore-defaults =
     .accesskey = R
 
 home-mode-choice-default-fx =
-    .label = { -firefox-home-brand-name } (Default)
+    .label = Homepage (Default)
 
 home-mode-choice-custom =
     .label = Custom URLs…
@@ -668,8 +670,8 @@ choose-bookmark =
 
 ## Home Section - Firefox Home Content Customization
 
-home-prefs-content-header2 = { -firefox-home-brand-name } Content
-home-prefs-content-description2 = Choose what content you want on your { -firefox-home-brand-name } screen.
+home-prefs-content-header2 = Homepage Content
+home-prefs-content-description2 = Choose what content you want on your Homepage screen.
 
 home-prefs-search-header =
     .label = Web Search
@@ -714,7 +716,7 @@ home-prefs-recent-activity-description = A selection of recent sites and content
 home-prefs-snippets-header =
     .label = Snippets
 
-home-prefs-snippets-description-new = Tips and news from { -vendor-short-name } and { -brand-product-name }
+home-prefs-snippets-description-new = Tips and news from { -vendor-short-name }
 
 # Variables:
 #   $num (number) - Number of rows displayed
