--- a/browser/base/content/browser.js
+++ b/browser/base/content/browser.js
@@ -1847,6 +1847,13 @@ var gBrowserInit = {
       )?.removeAttribute("key");
     }
 
+    if (Services.prefs.getBoolPref("librewolf.hidePasswdmgr", false)) {
+      PanelMultiView.getViewNode(
+        document,
+        "appMenu-passwords-button"
+      )?.remove();
+    }
+
     this._loadHandled = true;
   },
 
--- a/browser/components/preferences/privacy.js
+++ b/browser/components/preferences/privacy.js
@@ -810,6 +810,10 @@ var gPrivacyPane = {
     /* init HTTPS-Only mode */
     this.initHttpsOnly();
 
+    if (Services.prefs.getBoolPref("librewolf.hidePasswdmgr", false)) {
+      document.getElementById("passwordsGroup")?.remove();
+    }
+
     // Notify observers that the UI is now ready
     Services.obs.notifyObservers(window, "privacy-pane-loaded");
   },
