--- a/browser/components/preferences/privacy.inc.xhtml
+++ b/browser/components/preferences/privacy.inc.xhtml
@@ -54,7 +54,7 @@
         </description>
       </vbox>
     </hbox>
-    <vbox id="contentBlockingCategories">
+    <vbox id="contentBlockingCategories" hidden="true">
       <radiogroup id="contentBlockingCategoryRadio"
                   preference="browser.contentblocking.category"
                   aria-labelledby="trackingProtectionMenuDesc">
@@ -535,7 +535,7 @@
 <!-- History -->
 <groupbox id="historyGroup" data-category="panePrivacy" hidden="true">
   <label><html:h2 data-l10n-id="history-header"/></label>
-  <hbox align="center">
+  <hbox align="center" hidden="true">
     <label id="historyModeLabel"
            control="historyMode"
            data-l10n-id="history-remember-label"/>
@@ -585,8 +585,9 @@
         <vbox>
           <checkbox id="privateBrowsingAutoStart"
                     data-l10n-id="history-private-browsing-permanent"
-                    preference="browser.privatebrowsing.autostart"/>
-          <vbox class="indent">
+                    preference="browser.privatebrowsing.autostart"
+                    hidden="true" />
+          <vbox>
             <checkbox id="rememberHistory"
                       data-l10n-id="history-remember-browser-option"
                       preference="places.history.enabled"/>
@@ -1015,15 +1016,15 @@
 
 <!-- addons, forgery (phishing) UI Security -->
 <groupbox id="browsingProtectionGroup" data-category="panePrivacy" hidden="true">
-  <label><html:h2 data-l10n-id="security-browsing-protection"/></label>
-  <hbox align = "center">
+  <label><html:h2 data-l10n-id="security-browsing-protection" hidden="true"/></label>
+  <hbox align = "center" hidden="true">
     <checkbox id="enableSafeBrowsing"
               data-l10n-id="security-enable-safe-browsing"
               class="tail-with-learn-more"/>
     <label id="enableSafeBrowsingLearnMore"
            class="learnMore" is="text-link" data-l10n-id="security-enable-safe-browsing-link"/>
   </hbox>
-  <vbox class="indent">
+  <vbox class="indent" hidden="true">
     <checkbox id="blockDownloads"
               data-l10n-id="security-block-downloads"/>
     <checkbox id="blockUncommonUnwanted"
--- a/browser/locales/en-US/browser/preferences/preferences.ftl
+++ b/browser/locales/en-US/browser/preferences/preferences.ftl
@@ -1107,7 +1107,7 @@ addressbar-suggestions-settings = Change preferences for search engine suggestio
 
 content-blocking-enhanced-tracking-protection = Enhanced Tracking Protection
 
-content-blocking-section-top-level-description = Trackers follow you around online to collect information about your browsing habits and interests. { -brand-short-name } blocks many of these trackers and other malicious scripts.
+content-blocking-section-top-level-description = LibreWolf supports - and it enables by default - Enhanced Tracking Protection in Strict mode. This is one of the most important settings in the browser, as it provides state partitioning, strict blocking lists and other neat privacy features. We do not recommend changing to other modes.
 
 content-blocking-learn-more = Learn more
 
